diff --git a/src/vs/workbench/services/themes/common/workbenchThemeService.ts b/src/vs/workbench/services/themes/common/workbenchThemeService.ts
index 88deb0af..0f696041 100644
--- a/src/vs/workbench/services/themes/common/workbenchThemeService.ts
+++ b/src/vs/workbench/services/themes/common/workbenchThemeService.ts
@@ -39,8 +39,9 @@ export enum ThemeSettings {
 }

 export namespace ThemeSettingDefaults {
-	export const COLOR_THEME_DARK = 'Dark 2026';
-	export const COLOR_THEME_LIGHT = 'Light 2026';
+	// VS Sharp: default to bundled fogio JetBrains color theme.
+	export const COLOR_THEME_DARK = 'dark-jetbrains-color-theme';
+	export const COLOR_THEME_LIGHT = 'light-jetbrains-color-theme';
 	export const COLOR_THEME_HC_DARK = 'Default High Contrast';
 	export const COLOR_THEME_HC_LIGHT = 'Default High Contrast Light';

